R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -564,6 +566,26 @@ model <- spark.logit(df, Species ~ ., regParam = 0.056)
 summary(model)
 ```
 
+#### Linear Support Vector Machine (SVM) Classifier
+
+[Linear Support Vector Machine (SVM)](https://en.wikipedia.org/wiki/Support_vector_machine#Linear_SVM) classifier is an SVM classifier with linear kernel.
+This is a binary classifier. Multi-class classification can be achieved by one-vs-the-rest strategy. We use a simple example to show how to use `spark.svmLinear`
```

Comment:
we actually don't have support for `one-vs-the-rest strategy` in R at the moment (existing JIRA or design still open), so perhaps it's best we don't reference that here